drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -311,19 +311,18 @@ PodLoop:
 		containerPaths := getContainerPVCMountMap(*pod)
 		skipHostMountCheck := false
 		for containerName, paths := range containerPaths {
+			output, err := k8sCore.RunCommandInPod([]string{"cat", "/proc/mounts"}, pod.Name, containerName, pod.Namespace)
+			mounts := strings.Split(output, "\n")
```

Comment:
nit: Why did you move this line before checking the err? Makes more sense to do this only if the command was successful.

Reply:
good point! didn't do it intentionally, let me change it